### ISSUE TITLE
[BE - Requires Internal Engineer] Expense - Expense details disappear when opening expense after workspace is deleted

### ISSUE THREAD

**Issue by lanitochka17** [Expensify/App#51861]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Version Number:** 9.0.56-2
**Reproducible in staging?:** Y
**Reproducible in production?:** Y
**If this was caught on HybridApp, is this reproducible on New Expensify Standalone?:** N/A
**If this was caught during regression testing, add the test name, ID and link from TestRail:** N/A
**Email or phone of affected tester (no customers):** applausetester+kh2310001@applause.expensifail.com
**Issue reported by:** Applause - Internal Team

## Action Performed:
1. Go to staging.new.expensify.com
2. Create a new workspace
3. Go to workspace chat
4. Submit an expense
5. Go to Account settings > Workspaces
6. Delete the workspace
7. Go back to Inbox (on web, it will auto open the expense details)
8. (mobile only) open the workspace chat and then open the expense

## Expected Result:
Expense details will not disappear when opening expense after the workspace is deleted

## Actual Result:
Expense details disappear when opening expense after the workspace is deleted

## Workaround:
Unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [x] Android: Standalone
- [x] Android: HybridApp
- [x] Android: mWeb Chrome
- [x] iOS: Standalone
- [x] iOS: HybridApp
- [x] iOS: mWeb Safari
- [x] MacOS: Chrome / Safari
- [x] MacOS: Desktop

## Screenshots/Videos

<details>
  <summary>Add any screenshot/video evidence</summary>

  
</details> 

https://github.com/user-attachments/assets/387fb4cc-1885-463d-b0fe-139b4f6c68e8

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by FitseTLT** [comment]
This is a BE bug. On Open Report it is resetting the parent report id of the expense report.

**Comment by zanyrenney** [comment]
Thanks for figuring that out for us @FitseTLT

**Comment by zanyrenney** [comment]
waiting on internal engineers in quality for this

**Comment by zanyrenney** [comment]
Hmmm, maybe actually this is more fitting for expense!

**Comment by zanyrenney** [comment]
This is waiting on an engineer to pick up. It has been triaged but expense is not as high priority as our other waves. Changing to weekly for now.

**Comment by zanyrenney** [comment]
This is considered a bug/follow up. Seeing as expense is lower priority and there are a shortage of engineers to pick this up, this is still the appropriate label and prioritisation.

**Comment by zanyrenney** [comment]
Same as above, I think we can hold on this for now, keep it as weekly.

**Comment by zanyrenney** [comment]
I think Monthly is the right prioritisation.

**Comment by zanyrenney** [comment]
Waiting on internal capacity for this one.

**Comment by zanyrenney** [comment]
downgraded.

**Comment by zanyrenney** [comment]
I actually will retest again as this has been open a while.

**Comment by zanyrenney** [comment]
I can no longer reproduce this so I think it's fine to close this out!
